pkg/console/controllers/route/controller_test.go

```diff
@@ -200,29 +200,3 @@ func TestValidateCustomCertSecret(t *testing.T) {
 		})
 	}
 }
-
-func TestGetDefaultRouteHost(t *testing.T) {
```

Comment:
Could you explain the decision to remove these unit tests?

Reply:
It seems like the functionality may be checked in [these e2e tests](https://github.com/openshift/console-operator/pull/522/files#diff-5da9aaf9399e87599aa868fc1e2e371845bdde346e5642913e4c3b212d8fda5fR47), but I appreciate fail fast unit tests.

Reply:
Moved the function to a different pkg and forgot to move the tests... will update!